mParticle_Optimizely/MPKitOptimizely.m

```diff
@@ -241,12 +196,11 @@ - (MPKitExecStatus *)logEvent:(MPEvent *)event {
     
     NSDictionary *transformedEventInfo = [baseProductAttributes transformValuesToString];
     
-    [MPKitOptimizelyClient track:event.name
-                          userId:userId
-                      attributes:currentUser.userAttributes
-                       eventTags:transformedEventInfo];
+    [MPKitOptimizelyClient trackWithEventKey:event.name userId:userId attributes:currentUser.userAttributes eventTags:transformedEventInfo error:nil];
     
-    return [self execStatus:MPKitReturnCodeSuccess];
+    MPKitExecStatus *execStatus = [self execStatus:MPKitReturnCodeSuccess];
+    [execStatus forwardCount];
```

Comment:
Not sure what the intention here was. Maybe you were wanting to increment the forward count but it looks like it should be getting init'd to 1 by default in this case, also I think this would not be the way to do so.